split-sequence.asd

```diff
@@ -1,5 +1,7 @@
 ;;; -*- Lisp -*-
 
+(cl:in-package #:asdf-user)
```

Comment:
This is redundant because modern ASDF will evaluate the .asd file in a temporary package.